examples/basic/README.md

```diff
@@ -0,0 +1,38 @@
+### @artsy/fresnel
+
+# Basic Example
+
+Showcases the simplest possible use-case for `@artsy/fresnel`, and how one would use the lib when working on a typical client-side app.
+
+To run the example:
+
+```
+yarn install
+yarn start
+```
+
+#### Example Breakdown
+
+```js
+import { createMedia } from "@artsy/fresnel"
+
+const ExampleAppMedia = createMedia({
+  breakpoints: {
+    xs: 0,
+    sm: 768,
+    md: 1000,
+    lg: 1200,
+  },
+})
+
+const { Media, MediaContextProvider } = ExampleAppMedia
+
+const App = () => {
+  return (
+    <MediaContextProvider>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
+    </MediaContextProvider>
+  )
+}
+```
```

Comment:
Instead of repeating the README version and setting ourselves up for drift, perhaps the README example could link to this README and this README could link to the relevant source files, if even necessary?

Reply:
If anything, I think these sub-readme's should link back to the main readme. When I'm trying out a new lib (after researching, downloading, setting up, finding an issue, getting frustrated, repeat with something else) I want there to be, as immediately as possible, an example that I can copy and paste or at least _grok_ to see what I'm getting into before I repeate, yet again, another timely setup process. Since fresnel lends itself very well to such quick start examples -- using it is easy -- we'd be doing our users a disservice not showing them just how fast it can be to things up and running. From there we can dive into the details of the how and why. 

Side-note: I added updates to the main readme after I created these. Now that the main readme is updated I think these can be reduced to simple `yarn start` instructions.

Reply:
Actually, a nice middle-ground here is to Add a TOC at the top, with the `Overview` section just underneath. The Overview contains some light explaination and the basic example (similar to what's currently there). Basic example section goes away.